style.css

```diff
@@ -612,15 +630,143 @@ input[type="submit"] {
     width: 100%;
   }
 
-  .info {
-    background-image: url(Img/Header\ bg.png);
-    background-repeat: no-repeat;
-    background-size: 100% 100%;
-    top: 41px;
-    height: 100vh;
-  }
-
   .enlaces ul {
     width: 200px;
   }
+
+  .works {
+    grid-template-columns: 7% 86% 7%;
+  }
+
+  .card1 {
+    grid-area: 4/2/5/3;
+  }
+
+  .card2 {
+    grid-area: 5/2/6/3;
+    margin-bottom: 88px;
+  }
+
+  .card3 {
+    grid-area: 2/2/3/3;
+  }
+
+  .card4 {
+    grid-area: 3/2/4/3;
+    margin-bottom: 0;
+  }
+
+  .cg {
+    flex-direction: row;
+    gap: 17px;
+  }
+
+  .cg img {
+    width: 50%;
+  }
+
+  .c1dp {
+    font-size: 36px;
+    line-height: 52px;
+    color: #091e42;
+    width: 100%;
+  }
+
+  .li1 {
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .lisec {
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+    color: #6b778c;
+  }
+
+  .deskdisp img {
+    width: 8px;
+  }
+
+  .ca1p {
+    font-size: 16px;
+  }
+
+  .c1desc {
+    width: 100%;
+  }
+
+  .deskim {
+    display: flex;
+  }
+
+  .direc {
+    flex-direction: row-reverse;
+  }
+
+  .aboutinfo {
+    display: flex;
+    flex-direction: row;
+    padding-left: 0;
+    padding-right: 0;
+  }
```

Comment:
- [ ] You should stick to the design as much as possible like color, font, spacing, etc. In the About section, the left one takes 40% and the right one the remaining 60% of the page's width. Please do accordingly. Also please update the color of language, framework, and skills, the color use is `color: #344563;`.
- [ ] Also I would suggest please increase the box size for javascript, HTML and CSS and the spacing for it. Pleas see image below for reference
### Your design
![his](https://user-images.githubusercontent.com/81727988/236276196-be92298b-c67c-473a-8fbb-3736bc62456c.png)

### Figma design
![figma](https://user-images.githubusercontent.com/81727988/236276224-6a3ee8e0-8a94-4042-a186-5830e37706eb.png)

Reply:
@roshan-bajgain Also about the padding I removed those line of code and continue working with the last padding
![image](https://user-images.githubusercontent.com/119966922/236297891-712ba492-9105-47d5-bf97-40134507871b.png)
Is it ok?